src/foam/u2/wizard/analytics/AlternateFlowRefinement.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.analytics',
+  name: 'AlternateFlowRefinement',
+  refines: 'foam.u2.wizard.AlternateFlow',
+
+  imports: [
+    'analyticsAgent?'
+  ],
+
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'enableAnalytics'
+    },
+    {
+      class: 'String',
+      name: 'analyticsName'
+    }
+  ],
+
+  methods: [
+    {
+      name: 'refinedMethod_execute_AlternateFlowRefinement',
+      code: foam.u2.wizard.AlternateFlow.getAxiomByName('execute').code
+    },
```

Comment:
seems like this method is the old/original `execute` and the refinement of the `execute` function is the one below?

Reply:
Yes that's right. I decided to write this in a format that we might use if we ever add more convenient support for this in FOAM.

Reply:
so then maybe rename it eg. `execute0`, `executeOld` or just call `oam.u2.wizard.AlternateFlow.getAxiomByName('execute').code(x)` in the refinement method.

Reply:
What's wrong with this? I think this is more descriptive and will make it really easy to understand what's happening when looking at a stack trace

Reply:
> or just call oam.u2.wizard.AlternateFlow.getAxiomByName('execute').code(x) in the refinement method.
This will result in infinite recursion

Reply:
ah yes, it can't be in the refinement method itself

Reply:
Naming is hard! from the name `refinedMethod_execute_AlternateFlowRefinement` it doesn't tell that it's the old/original execute method

Reply:
that's what I would want a hypothetical FOAM feature for refining methods to call the original method internally, because in a stack trace it would be clearer that refinements has something to do with the method existing. Maybe `originalMethodBeforeRefinement_<name-of-method>_<name-of-refinement>` would be clearer.

Reply:
in u2/u3 we have `this.SUPER()` to call the parent method, maybe we can borrow the name for refinement to call the original method, what do you think?

Reply:
I think if it was ever added as a feature in FOAM both options would need to be available. something like `this.SUPER()` and `this.ORIGINAL()`